Editor/UI/Framework/SharedStyles.cs

```diff
@@ -131,5 +131,17 @@ public static void SetFontDynamicSize(int fontSize)
             LabelWithDynamicSizeWithDynamicSize.fontSize = fontSize;
             TextAreaWithDynamicSize.fontSize = fontSize;
         }
+
+        internal static Texture2D MakeColorTexture(Color col)
+        {
+            var pix = new Color[1];
+            pix[0] = col;
+
+            var result = new Texture2D(1, 1);
+            result.SetPixels(pix);
```

Comment:
Nit-picking minor optimization:

It looks like we can skip the Color array and set like this if it's just one pixel:  result.SetPixel(0, 0, col)